app/controllers/chat_controller.rb

```diff
@@ -329,6 +329,22 @@ def dismiss_retention_reminder
     render json: success_json
   end
 
+  def create_draft
```

Comment:
This is pretty nit-picky, but what do you think about renaming this action to `set_draft`, since it's used for creating _and_ destroying drafts?